MAINTAINERS.md

```diff
@@ -23,37 +23,22 @@ matching the repository name:
 This guide is intended for *maintainers* — anybody with commit access to one or more laminas Framework
 repositories.
 
-We use the [git flow methodology](http://nvie.com/posts/a-successful-git-branching-model/) for
+We use [automatic relases](https://github.com/laminas/automatic-releases) for
 managing the various laminas Framework repositories. At a glance, this means:
 
-- a **master** branch. This branch MUST be releasable at all times. Commits and merges against
-  this branch MUST contain only bugfixes and/or security fixes. Maintenance releases are tagged
-  against master.
-- a **develop** branch. This branch contains *new features*, and will either become the next minor
-  (feature) release or next major release. Typically, major releases are reserved for backwards
-  *incompatible* changes, but can also be used to signal major new features.
+- two or more minor version branches of the current major release
+  - the highest minor version branch must not be released (i.e. does not have matching version tags like `1.0.0` 
+    for `1.0.x`).
```

Comment:
This part isn't entirely true or necessary. There can be a single branch with no releases existing (e.g., a 0.1.x branch), or where the migration encountered a `develop` branch that had no changes from the `master` branch, and we only published the release branch associated with the most recent release.

AFTER a release has been made using the automatic-releases workflow, there will always be a minimum of two release branches. But BEFORE, there may only be one.